autoload/cmake4vim.vim

```diff
@@ -62,6 +62,9 @@ function! cmake4vim#GenerateCMake(...) abort
     " Prepare requests to CMake system
     call utils#cmake#common#makeRequests(l:build_dir)
 
+    " reload CMakeKits if needed
+    " call utils#cmake#reloadCMakeKits()
```

Comment:
I propose to remove unused code.

Reply:
This was for testing because this is the only change to `GenerateCMake` in this PR so I thought commenting this out might make CI work. But it did not help.